app/routes.js

```diff
@@ -1,7 +1,7 @@
-// const controller = require('./controllers/controller');
+const controller = require('./controllers/user.js');
```

Comment:
change the var name to something more declarative like `userController`. Also you don't need to add the `.js` in the filename.